test/puppetlabs/puppetdb/scf/storage_test.clj

```diff
@@ -608,14 +608,21 @@
                        "before"] nil})))
 
           (testing "should only delete the 1 edge"
```

Comment:
I wish there was some way we could test "this query does do a table scan" or similar, but I have no reasonable idea of how to implement taht.